### ISSUE TITLE
Support execution of multiple scripts with npm:build

### ISSUE THREAD

**Issue by rkrzewski** [allenai/sbt-plugins#194]
I'm using jspm and gulp for building my application's frontend. When the application is build from scratch (notably in CI build), `jspm install` needs to be performed before `gulp build` is started. I was fiddling around with different ways I could wire this together and the cleanest I was able to come up so far was declaring two separate `script` entries in `package.json` for the two tasks. 

When trying to integrate my gulp based build with SBT, I came across your plugin which I like a lot - it's orders of magnitude simpler that sbt-web ;) However, `npm:build` has a limitation that the invoked script is hardcoded to `build`. I was able to work around this by defining a third `script` entry in `package.json` that invokes a script that spawns two instances of `npm` in sequence, to execute the two other scripts. 

I'd like to propse a new setting for the plugin `nodeBuildScripts` of type `Seq[String]` with default value `Seq("build")`. `execBuild` method would then iterate over this sequence and run each of the requested scripts. The change would be fully backward compatible.

**Comment by markschaake** [comment]
@rkrzewski glad you are interested in using our plugins! You should be able to override the `build in Npm` task to do what you want it to do via something like:

```scala
// in build.sbt

NodeKeys.build in Npm := {
  // do stuff I need to happen before the default build task:
  ...
  // now run the previously set (default) build task:
  (NodeKeys.build in Npm).value
}
```

I recommend trying the above approach before issuing a PR. We use this plugin on several internal projects at AI2, and a change coming from an external PR would require someone on our team to do some manual testing before approval. This means it could take some time before someone can take a good look at the PR - probably not until Jan 2016. But please do issue a PR if you think it would be a better solution than what I've suggested above!

**Comment by rkrzewski** [comment]
January 2016 sounds great :) I have a JS based workaround right now, and secondly I am just prototyping a solution that would allow me to migrate a project from sbt-web to a native nodejs tooling.

**Comment by markschaake** [comment]
Resolved via #195
